setup/files.swr

```diff
@@ -53,6 +53,38 @@ folder InstallDir:\MSBuild\15.0\Bin\MSBuild
   file source=$(X86BinPath)Microsoft.Build.Core.xsd
   file source=$(X86BinPath)Microsoft.Build.CommonTypes.xsd
 
+folder InstallDir:\MSBuild\15.0\Bin\SdkResolvers\NuGet.MSBuildSdkResolver
+  file source=$(X86BinPath)NuGet.MSBuildSdkResolver.dll
+
+folder InstallDir:\MSBuild\15.0\Bin\SdkResolvers\NuGet.MSBuildSdkResolver\cs
+  file source=$(X86BinPath)cs\NuGet.MSBuildSdkResolver.resources.dll
+folder InstallDir:\MSBuild\15.0\Bin\SdkResolvers\NuGet.MSBuildSdkResolver\de
+  file source=$(X86BinPath)de\NuGet.MSBuildSdkResolver.resources.dll
+folder InstallDir:\MSBuild\15.0\Bin\SdkResolvers\NuGet.MSBuildSdkResolver\en
+  file source=$(X86BinPath)en\NuGet.MSBuildSdkResolver.resources.dll
+folder InstallDir:\MSBuild\15.0\Bin\SdkResolvers\NuGet.MSBuildSdkResolver\es
+  file source=$(X86BinPath)es\NuGet.MSBuildSdkResolver.resources.dll
+folder InstallDir:\MSBuild\15.0\Bin\SdkResolvers\NuGet.MSBuildSdkResolver\fr
+  file source=$(X86BinPath)fr\NuGet.MSBuildSdkResolver.resources.dll
+folder InstallDir:\MSBuild\15.0\Bin\SdkResolvers\NuGet.MSBuildSdkResolver\it
+  file source=$(X86BinPath)it\NuGet.MSBuildSdkResolver.resources.dll
+folder InstallDir:\MSBuild\15.0\Bin\SdkResolvers\NuGet.MSBuildSdkResolver\ja
+  file source=$(X86BinPath)ja\NuGet.MSBuildSdkResolver.resources.dll
+folder InstallDir:\MSBuild\15.0\Bin\SdkResolvers\NuGet.MSBuildSdkResolver\ko
+  file source=$(X86BinPath)ko\NuGet.MSBuildSdkResolver.resources.dll
```

Comment:
I think you have to address the x64 bin dir as well. One approach would be to just update the lines below than extract the nuget ones separately.

Reply:
The contents of the VSIX look correct

---
In reply to: [161095569](https://github.com/Microsoft/msbuild/pull/2855#discussion_r161095569) [](ancestors = 161095569)